### PR TITLE
fix: relative_path example

### PR DESCRIPTION
**Pull request by ericpaulsen**

### CODE REVIEW

**Comment by ericpaulsen** [comment]
@bpmct i see you fixed this in #22, yet it's not reflected on the site...

**Review by bpmct — APPROVED** [review]

File: docs/resources/app.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ resource "coder_app" "code-server" {
   name     = "VS Code"
   icon     = data.coder_workspace.me.access_url + "/icons/vscode.svg"
   url      = "http://localhost:13337"
-  path     = true
+  relative_path     = true
```
Comment (bpmct) [inline docs/resources/app.md:33]: @ericpaulsen can you adjust the formatting to align the = signs? `terraform lint` should achieve this

Comment (ericpaulsen) [reply docs/resources/app.md]: `terraform fmt` only applies to `.tf` files, so I manually updated the formatting.

**Comment by ericpaulsen** [comment]
@bpmct - also, could I get approved to merge PRs in this repo? thanks.

**Comment by bpmct** [comment]
I'm afraid I can't help with that cc @kylecarbs

**Comment by kylecarbs** [comment]
@ericpaulsen done

**Pull request merged**